galasa-maven-plugin/pom.xml

```diff
@@ -7,7 +7,7 @@
 	<groupId>dev.galasa</groupId>
 	<artifactId>galasa-maven-plugin</artifactId>
 	<packaging>maven-plugin</packaging>
-	<version>0.33.0</version>

```

Comment:
If we bump that, we need to bump the example output created by `galasactl project create` too.

Reply:
Does it get pulled-in to the obr or isolated also?